docfx.json

```diff
@@ -0,0 +1,46 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "files": [
+            "bin/**/*.dll"
+          ]
+        }
+      ],
+      "dest": "api",
+      "includePrivateMembers": false,
+      "disableGitFeatures": false,
+      "disableDefaultFilter": false,
+      "noRestore": false,
+      "namespaceLayout": "flattened",
+      "memberLayout": "samePage",
+      "EnumSortOrder": "alphabetic",
+      "allowCompilationErrors": false
+    }
+  ],
+  "build": {
+    "content": [
+      {
+        "files": [ "**/*.{md,yml}" ],
+        "exclude": [ "_site/**", "obj/**" ]
```

Comment:
`_site`? Do we have this folder?

Reply:
It will be auto-generated by the Github runner and copied to the gh-Pages branch. Then, Github will automatically deploy whatever is in the gh-pages branch. That's what I have setup in the .github/workflow.